src/metabase/driver/bigquery.clj

```diff
@@ -411,12 +406,16 @@
           :describe-table        (u/drop-first-arg describe-table)
           :details-fields        (constantly [{:name         "project-id"
                                                :display-name "Project ID"
-                                               :placeholder  "praxis-beacon-120871"
+                                               :placeholder  "bigquery-public-data"
                                                :required     true}
                                               {:name         "dataset-id"
                                                :display-name "Dataset ID"
-                                               :placeholder  "toucanSightings"
+                                               :placeholder  "baseball"
```

Comment:
I can change them back, I just put those in because they're a real public dataset someone could actually use.

I think it would be even better to have descriptions for the fields in the UI, sometimes it's difficult to figure out what the drivers are asking for. It's also not clear in the UI that `billing-project-id` is optional (and will default to `project-id`)